.gitignore

```diff
@@ -15,5 +15,9 @@ packages/lit-dev-api/lit/
 packages/lit-dev-api/api-data/**/*
 !packages/lit-dev-api/api-data/.gitkeep
 
+tests/**/*.png
```

Comment:
Why do we have this git pattern -- shouldn't the golden screenshots get checked in?

Reply:
Great question. Added this so we ignore any created locally. For example `-darwin.png`. This is paired with the gitignore beneath it that allows the `-linux.png` screenshots to be added.

Could make it more explicit with a single rule rather than two.

Reply:
Gotcha! Maybe just add a brief `#` comment about how we only want to keep linux screenshots since that's what we test on CI.